src/NETXUSASharp/Connector.cs

```diff
@@ -48,6 +48,20 @@ public TResponse Send<TResponse>(Enums.HttpVerbs method, string path)
         }
 
         /// <summary>
+        /// Combines a base URL and a path and returns a Uri
+        /// </summary>
+        /// <param name="url">Base URL optionally including a path.  E.g. http://rev.io/ or http://rev.io/directory/</param>
+        /// <param name="path">Relative path to be combined with URL</param>
+        /// <returns>System.Uri</returns>
+        public static Uri PathCombineUrl(string url, string path)
```

Comment:
I'll change it to private.  It was public to enable testing it from the Demo project but I tossed that code so it would be OK to change back.